### ISSUE TITLE
Add bandwidth throttling

### ISSUE THREAD

**Issue by mjaros** [vsespb/mt-aws-glacier#90]
Could this be implemented?

**Comment by vsespb** [comment]
I'll mark as enhancement. Plan to implement some day

**Comment by EQXTFL** [comment]
What about simply including http://lartc.org/wondershaper/

**Comment by vsespb** [comment]
Yes, there was a discussion already that that's what OS should do, but I think anyway, it worth implementing, as traffic shapers is hard to integrate. Traffic shaper is not something that can be used unix way: `mtglacier ... | shaper`, but intstead you need to know all DNS/ip of amazon and setup a separate service.

**Comment by michaelwittig** [comment]
+1

**Comment by ilijamt** [comment]
Any info on --max-hour-rate=100M suggestion in a previous issue?

**Comment by Farmer-Pete** [comment]
I've started using mt-glacier with `trickle`: http://monkey.org/~marius/pages/?page=trickle on Linux. It works well (as long as `--concurrency` is set to 1)

Example: `trickle -u 100 -d 100 -- mtglacier ...`
